README.md

```diff
@@ -19,6 +19,40 @@ cd ~/.
 git clone ssh://git@github.com/osrf/romi-dashboard
 cd romi-dashboard
 npm install
+
+```
+
+(Optional) Import external resources.
+
+```bash
+npm run setup
+```
+
+When executing this command, a prompt will be displayed, and it will guide you through the process.
+
+You have two options, import a specific folder with assets from a repository (like Github) or copy the assets folder from a local directory.
+
+In the case of choosing to obtain the data from a repository, you must define: (1) The repository URL and (2) the folder within the repository. In case of choosing to obtain the data from a local directory, the absolute path must be defined.
+
+This folder obtained must contain at its root a JSON file calling `main.json`, with the _RomiDashboard_ actors (robots, dispensers, etc.) and the path to their corresponding resources. E.g.:
+
+```json
+{
+  "robots":
+    "magni": {
+      "icons": {
+        "magni": "/robots/magni/magni.png"
```

Comment:
can we clarity if the key is the robot model or name?

Reply:
Makes sense! Thanks!